src/settings/config.js

```diff
@@ -0,0 +1,7 @@
+export const AppSettings = {
+    baseUrl: process.env.REACT_APP_BASE_URL,
```

Comment:
We are not able to use process.env in this current environment... Apologize for the lack of explanation - will add more to this comment soon when I have a few more minutes.

Reply:
kk sounds good, standing by :)